getting-started/vs-integration/new-project-wizard.md

```diff
@@ -10,13 +10,15 @@ position: 2
 
 This article demonstrates how to use the Telerik Visual Studio extensions to create a new project that is pre-configured for the Progress&reg; Telerik&reg; UI for Blazor components.
 
-To create a new Telerik UI for Blazor application, use the Create New Project wizard. The wizard detects all installed versions of Telerik UI for Blazor and lists them in the **Version** combobox&mdash;this feature enables you to start your project with the desired version. To make sure the components are up to date, you can also [get the latest version]({%slug getting-started-vs-integration-latest-version%}).
+To create a new Telerik UI for Blazor application, use the **Create New Project** wizard. The wizard detects all installed versions of Telerik UI for Blazor and lists them in the **Version** combobox to let you to start your project with the desired version. To make sure the components are up-to-date, you can also [get the latest version]({%slug getting-started-vs-integration-latest-version%}).
 
-The Create New Project Wizard automates a set of steps that you otherwise must perform manually.
+## How It Works
+
+The **Create New Project** wizard automates a set of steps that you otherwise must perform manually.
 
 The wizard adds the following client assets to your project:
-* [Telerik stylesheet]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files)
-* [JS Interop file]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files)
+* [Telerik stylesheet]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files).
+* [JS Interop file]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files).
 
 The wizard also configures the project to use the Telerik UI for Blazor components by adding the following utilities:
```

Comment:
The `TelerikLayout.razor` should highlight the TelerikRootComponent, rather than just mention that we are adding one more `razor` file. It's not immediately clear why we are doing this.